content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -64,18 +64,72 @@ Terraform Cloud has support for Sentinel's modules feature. This allows you to w
+		print("Details:", tz.text)
+		print("Abbreviation:", tz.abbr)
+		print("Offset:", tz.offset)
+		return tz.offset
+	}
+}
+```
 
--> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/foo.sentinel` will not work.
+The above configuration would tell a policy check to load the code at `./modules/timezone.sentinel` relative to the policy set working directory and make it available to be imported with the statement `import "timezone"` at the top of your Sentinel policy code. This module will be available to all of the policies within the policy set.
+
+-> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/timezone.sentinel` will not work.
 
 ### Sentinel policy code files
 
-Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `sunny-day.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set.
+Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `enforce-terraform-maintenance-windows.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set. 
+
+Using the `enforce-terraform-maintenance-windows.sentinel` policy as an example, we can use the `time` and `tfrun` imports along with our custom `timezone` module to enforce checks that:
+
+1. Load the time when the Terraform Run occurred
+1. Convert the loaded time to PST
+1. Verify that the provisioning operation is only going to occur on an agreed upon day
+
+An example policy would be as follows:
+
+```sentinel
+import "time"
+import "tfrun"
+import "timezone"
+
+param maintenance_days default ["Friday", "Saturday", "Sunday"]
+param timezone_abbreviation default "PST"
+
+tfrun_created_at = time.load(tfrun.created_at)
+
+supported_maintenance_day = rule {
+	tfrun_created_at.add(time.hour * timezone.offset(timezone_abbreviation)).weekday_name in maintenance_days
+}
+
+main = rule {
+	supported_maintenance_day
+}
+```
+
+-> **NOTE:** The above examples uses parameters to to facilitate module reuse within Terraform. For more information on parameters, see the [Sentinel parameter documentation](https://docs.hashicorp.com/sentinel/language/parameters/). We could expand the enforcement logic to also restrict provisioning to occur out of hours using the [time.hour](https://docs.hashicorp.com/sentinel/imports/time/#time-hour) function.
```

Comment:
```suggestion
For a more robust or flexible policy, we could expand the enforcement logic to also restrict provisioning to occur out of hours using the [time.hour](https://docs.hashicorp.com/sentinel/imports/time/#time-hour) function.

The above examples use parameters to to facilitate module reuse within Terraform. For more information on parameters, see the [Sentinel parameter documentation](https://docs.hashicorp.com/sentinel/language/parameters/). 
```

I think this can just be paragraphs instead of a blue box. 

I switched the order here to make a smoother transition out of the example. (Ponder fiddly time math -> consider other bits of fiddly time math that might be possible -> dump fiddly time math out of brain's cache -> move on to thinking about structural issues and generalities again.)